pkgs/by-name/gr/gradia/package.nix

```
@@ -0,0 +1,64 @@
+{
+  lib,
+  gtk4,
+  glib,
+  ninja,
+  meson,
+  stdenv,
+  python3,
+  libportal,
+  libadwaita,
+  wrapGAppsHook4,
+  fetchFromGitHub,
+  nix-update-script,
+  blueprint-compiler,
+  desktop-file-utils,
+  gobject-introspection,
+}:
+stdenv.mkDerivation (finalAttrs: {
```

```suggestion
python3Packages.buildPythonApplication {
  format = "other";
```
This almost certainly should be packaged as a python application!
If you set `"other"` format, `buildPythonApplication`can use meson too.
See e.g. the `refine` package for example: https://github.com/NixOS/nixpkgs/blob/0e9e520ad13e41b04e90db358f0bba0970c42180/pkgs/by-name/re/refine/package.nix#L1

---

Done!